README.md

```diff
@@ -114,15 +114,44 @@ If you experience errors, after a version update, try a fresh install:
 
 Runs code quality checks, generates prebid.js files and creates zip archive distributable:
 
-   `./build/dev/prebid.js` Full source code for dev and debug
-    `./build/dev/prebid.js.map` Source map for dev and debug
-    `./build/dist/prebid.js` Minified production code
-    `./prebid.js_<version>.zip` Distributable
+- `./build/dev/prebid.js` Full source code for dev and debug
+- `./build/dev/prebid.js.map` Source map for dev and debug
+- `./build/dist/prebid.js` Minified production code
+- `./prebid.js_<version>.zip` Distributable
 
 Code quality is defined by `./.jscs` and `./.jshint` files and errors are reported in the
 terminal. The build will continue with quality errors, however. If you are contributing code
 you can configure your editor with the provided .jscs and .jshint settings.
 
+### Optimisation ###
```

Comment:
Looks good. Could you break these into 2 sections maybe? I'm thinking something like 1) Build standalone prebid.js 2) Build using NPM for bundling